### ISSUE TITLE
[Bug]: external WFS - some problems

### ISSUE THREAD

**Issue by josemvm** [3liz/lizmap-web-client#3886]
### What is the bug?

for wfs layer:
- the atribbute table shows the columns but not the values
- the atribbute table shows nothing if the user set "limitDataToBbox"
- the popup shows nothing

### Steps to reproduce the issue

please try it

### Versions

**Versions :**

* Lizmap Web Client : 3.6.5
* Lizmap plugin : 3.17.0
* QGIS Desktop : 3.22.16
* QGIS Server : 3.22.16
* Py-QGIS-Server : not used
* QGIS Server plugin atlasprint : 3.3.2
* QGIS Server plugin lizmap_server : 2.8.0
* QGIS Server plugin wfsOutputExtension : 1.8.0

### Check Lizmap plugin

- [X] I have done the step just before in the Lizmap QGIS desktop plugin before opening this ticket. Otherwise, my ticket is not considered valid and might get closed.

### QGIS server version, only if the section above doesn't mention the QGIS Server version

3.22.16

### Operating system

Ubuntu 20.04

### Browsers

Firefox

### Browsers version

Firefox 117.0.1

**Comment by josemvm** [comment]
often the layer map does not appear...

**Comment by gioman** [comment]
@josemvm any public URL we can test?

**Comment by josemvm** [comment]
everything works fine and speedly in qgis desktop

**Comment by gioman** [comment]
> everything works fine and speedly in qgis desktop

@josemvm I think you should test/try the WMS/WFS service generated by QGIS server that is cascading that services (not the one generated by LMWC, just QGIS server) and see if it works as expected. 

Is this ticket different than https://github.com/3liz/lizmap-web-client/issues/3884 ?

**Comment by Gustry** [comment]
> everything works fine and speedly in qgis desktop

I guess you are talking about the external WFS straight in QGIS desktop.
Indeed, if you can try in QGIS desktop : 
* the WFS from QGIS server
* the WFS from LWC

**Comment by josemvm** [comment]
>Is this ticket different than https://github.com/3liz/lizmap-web-client/issues/3884 ?

yes, this ticket https://github.com/3liz/lizmap-web-client/issues/3884 ? is only about authentication for the same wfs service

**Comment by josemvm** [comment]
> @josemvm I think you should test/try the WMS/WFS service generated by QGIS server that is cascading that services (not the one generated by LMWC, just QGIS server) and see if it works as expected.

how can i test it?...

**Comment by josemvm** [comment]
now i'm using WFS service generated by QGIS Server, cascading the layer from original source/wfs service

on QGIS Desktop i' ve create a wfs connectiona to WFS service generated by QGIS Server and once again everything works fine on QGIS Desktop

when i publish the project on LWC it doesn't really work, so it's a LWC problem and not QGIS Server one

**Comment by josemvm** [comment]
however the map works fine on QGIS Desktop but de QGIS Server doesn´t cascade the table values only the table columns with empty values

**Comment by Gustry** [comment]
I took the the WFS from 

https://demo.snap.lizmap.com/lizmap_3_6/index.php/view/map?repository=features&project=fire_hydrant_actions

and publish into 

https://demo.snap.lizmap.com/lizmap_3_7/index.php/view/map?repository=debug&project=external_wfs

by changing the color of the buildings (to check it's WFS)

it works, and attribute table as well.
The attribute table looks slow to open, but it works with this layer.

<details>

<summary>Versions of snap 3.7 server</summary>

* Lizmap Web Client : 3.7.0-alpha.3
* Lizmap plugin : master
* QGIS Desktop : 3.28.11
* Lizmap.com : Yes
* QGIS Server : 3.28.11
* Py-QGIS-Server : 1.8.8
* QGIS Server plugin atlasprint : 3.3.3-alpha
* QGIS Server plugin cadastre : 1.17.4-alpha
* QGIS Server plugin lizmap_server : 2.8.1-alpha
* QGIS Server plugin wfsOutputExtension : 1.8.1-alpha
</details>

**Comment by Gustry** [comment]
> we are working in different contexts:

I'm not sure to understand what you mean. At some point, the WFS relies on a datasource. The project `external_wfs.qgs` project is indeed using an external WFS, same as your question.

> sometimes (often) the map doesn't appear at its initial extent, nor with zoom out and zoom in

Indeed, strange now.
